### PR TITLE
Fix formatting

### PR DESCRIPTION
**Pull request by luanpotter**
Add scripts for fix formatting and formatting check & fix existing code.

### CODE REVIEW

**Review by erickzanardo — APPROVED** [review]
LGTM

File: scripts/lint.sh [hunk 1]
```diff
@@ -0,0 +1,16 @@
+#!/bin/bash -e
+
+if [[ $(flutter format -n .) ]]; then
+    echo "flutter format issue"
```
Comment (erickzanardo) [inline scripts/lint.sh:4]: We should echo the error here as well.

Comment (luanpotter) [reply scripts/lint.sh:4]: it's not very relevant because you will just use the fix script to fix right?

this is the script we use in all other repos

Comment (renancaraujo) [reply scripts/lint.sh:4]: the flutter format output is pretty useless.

**Pull request merged**